server/sonar-server/src/main/java/org/sonar/server/rule/ws/CreateAction.java

```diff
@@ -62,19 +63,22 @@
   private final DbClient dbClient;
   private final RuleCreator ruleCreator;
   private final RuleMapper ruleMapper;
+  private final OrganizationFlags organizationFlags;
 
-  public CreateAction(DbClient dbClient, RuleCreator ruleCreator, RuleMapper ruleMapper) {
+  public CreateAction(DbClient dbClient, RuleCreator ruleCreator, RuleMapper ruleMapper, OrganizationFlags organizationFlags) {
     this.dbClient = dbClient;
     this.ruleCreator = ruleCreator;
     this.ruleMapper = ruleMapper;
+    this.organizationFlags = organizationFlags;
   }
 
   @Override
   public void define(WebService.NewController controller) {
     WebService.NewAction action = controller
       .createAction("create")
       .setDescription("Create a custom rule. <br/>" +
-        "Since 5.5, it's no more possible to create manual rule.")
+        "Since 5.5, it's no more possible to create manual rule." +
+        "Custom rules are currently not supported, if the organization feature is enabled. In that case, the webservice will fail.")
```

Comment:
I would say "Custom rules are not supported if the organization feature is enabled.".